action.yml

```diff
@@ -1,36 +1,43 @@
-name: 'Upload files to a GitHub release'
-description: 'Upload files to a GitHub release (cross-platform)'
-author: 'Sven-Hendrik Haase'
+name: "Upload files to a GitHub release"
+description: "Upload files to a GitHub release (cross-platform)"
+author: "Sven-Hendrik Haase"
 branding:
   icon: archive
   color: orange
 inputs:
   repo_token:
-    description: 'GitHub token.'
+    description: "GitHub token."
```

Comment:
Please make the linting-type changes in a separate PR in order to minimize the changes in this one.